site/pages/components/image.tsx

```diff
@@ -0,0 +1,531 @@
+import React from 'react';
+import {InlineMessage, toNewsKitIcon, UnorderedList, Image} from 'newskit';
+import {Info as FilledInfo} from '@emotion-icons/material/Info';
+import {ComponentPageTemplate} from '../../templates/component-page-template/component-page-template';
+import {getIllustrationComponent} from '../../components/illustrations/illustration-loader';
+import {LayoutProps} from '../../components/layout';
+import {MetaStatus} from '../../components/meta/types';
+import {Link} from '../../components/link';
+import {UsageKind} from '../../components/usage-card';
+import {IconFilledCircle} from '../../components/icons';
+import {InlineCode} from '../../components/markdown-elements';
+import {ContentText} from '../../components/text-section/content-text';
+
+const IconFilledInfo = toNewsKitIcon(FilledInfo);
+
+const infoIcon = (
+  <IconFilledInfo
+    overrides={{
+      size: 'iconSize020',
+    }}
+  />
+);
+
+const ImageComponent = (layoutProps: LayoutProps) => (
+  <ComponentPageTemplate
```

Comment:
You can check how we started typing this template for example in Slider page.
```suggestion
  <ComponentPageTemplate<typeof Image>
```

Reply:
@mutebg will check -thanks!